src/Console/Command/SelfUpdateCommand.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Pickle\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Humbug\SelfUpdate\Updater;
+
+class SelfUpdateCommand extends Command
+{
+    const PHAR_NAME = 'pickle.phar';
+
+    protected function configure()
+    {
+        $this
+            ->setName('self-update')
+            ->setDescription('Validate a PECL extension')
+            ->addOption(
+                'unstable',
+                null,
+                InputOption::VALUE_NONE,
+                'Update to a non-stable version'
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if (\Phar::running() === '') {
+            throw new \RuntimeException($this->getName() . ' can only be run from the Phar!');
+        }
+
+        $updater = new Updater(null, false, Updater::STRATEGY_GITHUB);
+        $updater->getStrategy()->setPackageName('friendsofphp/pickle');
+        $updater->getStrategy()->setPharName(self::PHAR_NAME);
+        $updater->getStrategy()->setCurrentLocalVersion('v0.5.0');
```

Comment:
It's might be worth moving this somewhere more obvious. Don't forget to update this when tagging... Any way of automating it using the PHAR build process?